docs/source/user/settings.rst

```diff
@@ -70,6 +70,20 @@ overwhelming users with too many options*.
 If for some reason you need to enable direct editing of user group items
 you can do so by setting this to ``True``.
 
+``OPENWISP_RADIUS_USERGROUP_ADMIN``
```

Comment:
Shouldn't this be `OPENWISP_RADIUS_USER_ADMIN_RADIUSTOKEN_INLINE`?

Why `OPENWISP_RADIUS_USERGROUP_ADMIN`?